src/OpenTelemetry.Exporter.OpenTelemetryProtocol/CHANGELOG.md

```diff
@@ -10,6 +10,14 @@
   are now included in this package.
   ([#4556](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4556))
 
+* Added support for the OTLP gRPC exporter to retry failed requests due to
```

Comment:
This doesn't cover `PartialSuccess`, right? We wouldn't be retrying any of the rejected items from a partially successful response.

Reply:
No, the spec does not support retrying rejected items from a partially successful response. However, you bring up a good point, the spec does say

> The client MUST NOT retry the request when it receives a partial success response where the partial_success is populated.

I need to fix this.